Discussion/Discussion/Presentation/Comments/Base/CommentCell.swift

```diff
@@ -68,15 +68,15 @@ public struct CommentCell: View {
                     onReportTap()
                 }, label: {
                     comment.abuseFlagged
-                    ? CoreAssets.reported.swiftUIImage
-                    : CoreAssets.report.swiftUIImage
+                    ? (CoreAssets.reported.swiftUIImage.renderingMode(.template))
```

Comment:
do we need outer brackets here?

Reply:
Without these brackets the Xcode is generating the following warning `Void Function in Ternary Violation: Using ternary to call Void functions should be avoided (void_function_in_ternary)`

Reply:
addressed!